report/src/main/java/com/hartwig/actin/report/pdf/tables/treatment/EligibleActinTrialsGenerator.java

```diff
@@ -56,7 +56,7 @@ public static EligibleActinTrialsGenerator forClosedTrials(@NotNull List<Evaluat
             }
         }
 
-        String title = String.format("%s trials that are closed or blacklisted but considered eligible (%s)",
+        String title = String.format("%s trials and cohorts that are considered ineligible (%s)",
```

Comment:
This looks like an accidental change, since we still display trials that are closed or blacklisted but considered eligible?